mpas_analysis/ocean/sst_timeseries.py

```diff
@@ -29,47 +39,59 @@ def sst_timeseries(config):
     # reading only those that are between the start and end dates
     startDate = config.get('time', 'timeseries_start_date')
     endDate = config.get('time', 'timeseries_end_date')
-    infiles = streams.readpath('timeSeriesStatsOutput',
-                               startDate=startDate, endDate=endDate)
-    print 'Reading files {} through {}'.format(infiles[0],infiles[-1])
-
+    streamName = streams.find_stream(streamMap['timeSeriesStats'])
+    infiles = streams.readpath(streamName, startDate=startDate,
+                               endDate=endDate)
+    print 'Reading files {} through {}'.format(infiles[0], infiles[-1])
 
-    casename = config.get('case','casename')
-    ref_casename_v0 = config.get('case','ref_casename_v0')
-    indir_v0data = config.get('paths','ref_archive_v0_ocndir')
+    casename = config.get('case', 'casename')
+    ref_casename_v0 = config.get('case', 'ref_casename_v0')
+    indir_v0data = config.get('paths', 'ref_archive_v0_ocndir')
 
-    plots_dir = config.get('paths','plots_dir')
+    plots_dir = config.get('paths', 'plots_dir')
 
-    yr_offset = config.getint('time','yr_offset')
+    yr_offset = config.getint('time', 'yr_offset')
 
-    N_movavg = config.getint('sst_timeseries','N_movavg')
+    N_movavg = config.getint('sst_timeseries', 'N_movavg')
 
     regions = config.getExpression('regions', 'regions')
     plot_titles = config.getExpression('regions', 'plot_titles')
     iregions = config.getExpression('sst_timeseries', 'regionIndicesToPlot')
 
     # Load data:
-    ds = xr.open_mfdataset(infiles, preprocess=lambda x: 
-                           preprocess_mpas(x, yearoffset=yr_offset,
-                                           timestr=['xtime_start', 'xtime_end'],
-                                           onlyvars=['time_avg_avgValueWithinOceanRegion_avgSurfaceTemperature']))
+    varList = ['avgSurfaceTemperature']
```

Comment:
I personally don't prefer such short names.  In fact, I may do a separate PR to rename `sst` and `mld` something more descriptive, especially since `sst` actually refers to the full temperature, which is later indexed at the surface.  But that's not something for this PR.